packages/snip20/src/handle.rs

```diff
@@ -0,0 +1,562 @@
+use serde::Serialize;
+
+use cosmwasm_std::{to_binary, Binary, Coin, CosmosMsg, HumanAddr, StdResult, Uint128, WasmMsg};
+
+use secret_toolkit_utils::space_pad;
+
+/// SNIP20 token handle messages
+#[derive(Serialize)]
+#[serde(rename_all = "snake_case")]
+pub enum HandleMsg<'a> {
```

Comment:
@reuvenpo The reason I was only deriving Serialize for the Handle/QueryMsg was that I was anticipating that their usefulness would be short-lived.  Since their only real value is to generate the message, I thought that the flow would probably be 1. Create the HandleMsg variant instance, 2. generate the CosmosMsg by calling `to_cosmos_msg`, 3. Never touch it again.  Or for QueryMsg, 1.Create the variant instance, 2. Call `query`, 3. Never touch QueryMsg again, but use whatever type was returned form the query.  That's why the query return types all have numerous `derive`s but the enums that are just used to create the messages and thrown away are minimal.  (Obviously if they are calling the individually-named functions, they just get created, used, and released.)
Do you think people might have more use for the Handle/QueryMsg enums later on?  If so I can just add a bunch of `derive`s